os/sdk-modifying-coreos.md

```diff
@@ -168,7 +168,19 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
-If you encounter errors with KVM, verify that it is supported by your CPU and enabled in the BIOS.
+If you encounter errors with KVM, verify that virtualization is supported by your CPU by running `egrep '(vmx|svm)' /proc/cpuinfo`. The `/dev/kvm` directory will be in your host OS when virtualization is enabled in the BIOS.
+
+The `./coreos_production_qemu.sh` file can be found in the `~/trunk/src/build/images/amd64-usr/latest` directory inside the SDK chroot.
+
+#### Boot Options
+
+After `image_to_vm.sh` completes, run `./coreos_production_qemu.sh -curses` to launch a graphical interface to log in to the Container Linux VM.
+
+You could instead use the `-nographic` option, `./coreos_production_qemu.sh -nographic` , which gives you the ability to switch from the VM to the QEMU monitor console by pressing <kbd>CTRL</kbd>+<kbd>a</kbd> and then <kbd>c</kbd>. To exit out of the Container Linux log in prompt, press <kbd>CTRL</kbd>+<kbd>a</kbd> and then <kbd>x</kbd>.
+
+You could also log in via SSH by running `./coreos_production_qemu.sh` and then run `ssh core@127.0.0.1 -p 2222` to enter the guest OS. Alternatively, you can log in via SSH keys by running `./coreos_production_qemu.sh -a ~/.ssh/authorized_keys -p 2223 -- -curses`. Refer to the [Booting with QEMU](booting-with-qemu.md#SSH-keys) guide for more information on this usage.
```

Comment:
`and then running`

As I said before, `-a ~/.ssh/authorized_keys -p 2223` shouldn't be necessary here.  Are you seeing otherwise?

Reply:
My intention was to include the suggestions that show up after `./image_to_vm` is done running: 
![screenshot from 2018-06-06 12-22-00](https://user-images.githubusercontent.com/12985181/41061168-9a059096-6986-11e8-8e4a-ce0a68dca31a.png)

I've rephrased the "Alternatively..." sentence